zio-http/src/main/scala/zhttp/http/Request.scala

```diff
@@ -102,6 +102,9 @@ trait Request extends HeaderExtension[Request] { self =>
    */
   def setUrl(url: URL): Request = self.copy(url = url)
 
+  override def toString: String =
+    s"Request($method, $url, $headers, $remoteAddress)"
```

Comment:
Small change required, we added version information in the Request. Let's add that too in this method

Reply:
Done.